recipes/logr/all/conanfile.py

```diff
@@ -13,110 +20,97 @@ class LogrConan(ConanFile):
         "for server/desktop applications"
     )
     topics = ("logger", "development", "util", "utils")
-    generators = "cmake"
+
     settings = "os", "compiler", "build_type", "arch"
-    no_copy_source = True
 
     options = {"backend": ["spdlog", "glog", "log4cplus", "boostlog", None]}
     default_options = {"backend": "spdlog"}
 
-    _cmake = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
+    def layout(self):
+        basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("fmt/8.1.1")
+        if Version(self.version) >= "0.6.0":
+            fmt_ref = "fmt/9.1.0"
+            spdlog_ref = "spdlog/1.11.0"
+        else:
+            fmt_ref = "fmt/8.1.1"
+            spdlog_ref = "spdlog/1.9.2"
+
+        self.requires(fmt_ref)
 
         if self.options.backend == "spdlog":
-            self.requires("spdlog/1.9.2")
+            self.requires(spdlog_ref)
         elif self.options.backend == "glog":
-            self.requires("glog/0.5.0")
+            self.requires("glog/0.6.0")
         elif self.options.backend == "log4cplus":
             self.requires("log4cplus/2.0.5")
         elif self.options.backend == "boostlog":
             self.requires("boost/1.77.0")
 
-    def configure(self):
+    def package_id(self):
+        self.info.settings.clear()
+        self.info.requires.clear()
+        self.info.options.backend
```

Comment:
```suggestion
        del self.info.options.backend
```

Reply:
Well, I guess I need it and the reason is the following.

Installing with cmake copies only a single backend header file (dedicated to this backend). Without this I will have install and have only a default backend header and the same dir would be promoted even if I want a package with a different backend.

So I would like a package_id be different for different backends, while things like build_type, compiler, etc doesn't matter.

Reply:
and it's the only option? so that easy

That line does not actually do anything the default is to "factor everything in" so this will just keep options which the only one is backend :)

```suggestion
```

Reply:
Works like a charm, thanks!